crates/matrix-sdk-base/src/event_cache/store/media/integration_tests.rs

```diff
@@ -12,26 +12,29 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Trait and macro of integration tests for `EventCacheStoreMedia`
+//! Trait and macro of integration tests for `MediaStoreInner`
 //! implementations.
 
 use ruma::{
     events::room::MediaSource,
+    media::Method,
     mxc_uri, owned_mxc_uri,
     time::{Duration, SystemTime},
+    uint,
 };
 
-use super::{
-    EventCacheStoreMedia, MediaRetentionPolicy, media_service::IgnoreMediaRetentionPolicy,
+use super::{MediaRetentionPolicy, MediaStoreInner, media_service::IgnoreMediaRetentionPolicy};
+use crate::{
+    event_cache::store::media::media_service::MediaStore,
+    media::{MediaFormat, MediaRequestParameters, MediaThumbnailSettings},
 };
-use crate::media::{MediaFormat, MediaRequestParameters};
 
-/// [`EventCacheStoreMedia`] integration tests.
+/// [`MediaStoreInner`] integration tests.
 ///
 /// This trait is not meant to be used directly, but will be used with the
-/// `event_cache_store_media_integration_tests!` macro.
+/// `media_store_inner_integration_tests!` macro.
 #[allow(async_fn_in_trait)]
-pub trait EventCacheStoreMediaIntegrationTests {
+pub trait MediaStoreInnerIntegrationTests {
```

Comment:
Can we move that in `crates/matrix-sdk-base/src/media/store/integration_tests.rs`?